Sources/NIOTransportServices/NIOTSNetworkEvents.swift

```diff
@@ -106,6 +119,8 @@ extension NIOTSNetworkEvents.BetterPathAvailable: Sendable {}
 @available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
 extension NIOTSNetworkEvents.BetterPathUnavailable: Sendable {}
 @available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
+extension NIOTSNetworkEvents.ViabilityUpdate: Sendable {}
```

Comment:
This extension is always `Sendable`, so it doesn't need the `@available`.

Reply:
Isn't NIOTSNetworkEvents limited to `@available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)`

Reply:
Any suggestions? @Lukasa

Reply:
@Cartisim you are correct here, the availability is required because of `NIOTSNetworkEvents`